### PR TITLE
[zh] update dns-pod-server.md

### PR DESCRIPTION
**Pull request by TinySong**
Signed-off-by: song <tinysong1226@gmail.com>

### CODE REVIEW

File: content/zh/docs/concepts/services-networking/dns-pod-service.md [hunk 1]
```diff
@@ -565,6 +565,41 @@ a list of search domains of up to 2048 characters.
...
+  and `kubernetes`, but not the partially qualified names (`kubernetes.defaul
+  `kubernetes.default.svc`).
+-->
+## Windows 节点上的 DNS 解析 {#dns-windows}
+
+- Windows 节点运行的 Pod 不支持 ClusterFirstWithHostNet。
+Windows 将所有带有 `.` 的名称视为全限定域名（FQDN）并跳过全限定域名（FQDN）解析。
+- 在 Windows 上，可以使用的 DNS 解析器有很多。
+由于这些解析器彼此之间会有轻微的行为差别，
+建议使用[`Resolve-DNSName`](https://docs.microsoft.com/powershell/module/dnsclient/resolve-dnsname)
```
Comment (Sea-n) [inline content/zh/docs/concepts/services-networking/dns-pod-service.md:593]:
```suggestion
建议使用 [`Resolve-DNSName`](https://docs.microsoft.com/powershell/module/dnsclient/resolve-dnsname)
```

File: content/zh/docs/concepts/services-networking/dns-pod-service.md [hunk 2]
```diff
@@ -565,6 +565,41 @@ a list of search domains of up to 2048 characters.
...
+powershell cmdlet 进行名称查询解析。
+- 在 Linux 上，有一个 DNS 后缀列表，当解析全名失败时可以使用。
+在 Windows 上，你只能有一个 DNS 后缀,
```
Comment (Sea-n) [inline content/zh/docs/concepts/services-networking/dns-pod-service.md:596]:
```suggestion
在 Windows 上，你只能有一个 DNS 后缀，
```

File: content/zh/docs/concepts/services-networking/dns-pod-service.md [hunk 3]
```diff
@@ -565,6 +565,41 @@ a list of search domains of up to 2048 characters.
...
+及与该 Pod 的命名空间相关联的 DNS 后缀（例如：`mydns.svc.cluster.local`）。
+Windows 可以解析全限定域名（FQDN），和使用了该 DNS 后缀的服务名称或者网络名称。
```
Comment (Sea-n) [inline content/zh/docs/concepts/services-networking/dns-pod-service.md:598]: Ref: [Wiki FQDN](https://zh.wikipedia.org/zh-tw/%E5%AE%8C%E6%95%B4%E7%B6%B2%E5%9F%9F%E5%90%8D%E7%A8%B1)

```suggestion
Windows 可以解析絕對網域名稱（FQDN）、使用了该 DNS 后缀的服务名称，或者网络名称。
```

File: content/zh/docs/concepts/services-networking/dns-pod-service.md [hunk 4]
```diff
@@ -565,6 +565,41 @@ a list of search domains of up to 2048 characters.
...
+例如， 在 `default` 命名空间中生成一个 Pod，该 Pod 会获得的 DNS 后缀为 `default.svc.cluster.local`。
+在 Windows 的 Pod 中，你可以解析 `kubernetes.default.svc.cluster.local` 和 `kubernetes`，
+但是不能解析部分限定名称（`kubernetes.default` 和 `kubernetes.default.svc`）。
```
Comment (Sea-n) [inline content/zh/docs/concepts/services-networking/dns-pod-service.md:601]: Not sure `部分域名` or `相對域名` is better.
```suggestion
但是不能解析相對網域名稱（`kubernetes.default` 和 `kubernetes.default.svc`）。
```

File: content/zh/docs/concepts/services-networking/dns-pod-service.md [hunk 5]
```diff
@@ -565,6 +565,41 @@ a list of search domains of up to 2048 characters.
...
+Windows 将所有带有 `.` 的名称视为全限定域名（FQDN）并跳过全限定域名（FQDN）解析。
+- 在 Windows 上，可以使用的 DNS 解析器有很多。
+由于这些解析器彼此之间会有轻微的行为差别，
+建议使用[`Resolve-DNSName`](https://docs.microsoft.com/powershell/module/dnsclient/resolve-dnsname)
+powershell cmdlet 进行名称查询解析。
+- 在 Linux 上，有一个 DNS 后缀列表，当解析全名失败时可以使用。
+在 Windows 上，你只能有一个 DNS 后缀,
+及与该 Pod 的命名空间相关联的 DNS 后缀（例如：`mydns.svc.cluster.local`）。
+Windows 可以解析全限定域名（FQDN），和使用了该 DNS 后缀的服务名称或者网络名称。
+例如， 在 `default` 命名空间中生成一个 Pod，该 Pod 会获得的 DNS 后缀为 `default.svc.cluster.local`。
```
Comment (Sea-n) [inline content/zh/docs/concepts/services-networking/dns-pod-service.md:599]:
```suggestion
例如，在 `default` 命名空间中生成一个 Pod，该 Pod 会获得的 DNS 后缀为 `default.svc.cluster.local`。
```

File: content/zh/docs/concepts/services-networking/dns-pod-service.md [hunk 6]
```diff
@@ -565,6 +565,41 @@ a list of search domains of up to 2048 characters.
...
+  or network name which can be resolved with this single suffix. For example, a pod spawned
+  in the `default` namespace, will have the DNS suffix `default.svc.cluster.local`.
+  Inside a Windows pod, you can resolve both `kubernetes.default.svc.cluster.local`
+  and `kubernetes`, but not the partially qualified names (`kubernetes.defaul
+  `kubernetes.default.svc`).
+-->
+## Windows 节点上的 DNS 解析 {#dns-windows}
+
+- Windows 节点运行的 Pod 不支持 ClusterFirstWithHostNet。
+Windows 将所有带有 `.` 的名称视为全限定域名（FQDN）并跳过全限定域名（FQDN）解析。
```
Comment (Sea-n) [inline content/zh/docs/concepts/services-networking/dns-pod-service.md:590]: If the content is part of list item, please include two leading spaces.

```suggestion
- Windows 节点运行的 Pod 不支持 ClusterFirstWithHostNet。
  Windows 将所有带有 `.` 的名称视为全限定域名（FQDN）并跳过全限定域名（FQDN）解析。
```

File: content/zh/docs/concepts/services-networking/dns-pod-service.md (hunk 3 above)
Comment (howieyuen) [reply content/zh/docs/concepts/services-networking/dns-pod-service.md:598]: I think "全限定域名" is preferable for simple Chinese, cause we always call it like this.
and how about translating like this ?
```suggestion
Windows 可以解析全限定域名（FQDN）、服务或网络名称，它们可以使用此单个后缀进行解析。
```

Comment (tengqm) [reply content/zh/docs/concepts/services-networking/dns-pod-service.md:598]: 现在的翻译是可接受的。

**Comment by Sea-n** [comment]
This PR seems to be conflicted with #33993, maybe you can check if this file have other place need to update.
- #33993

File: content/zh/docs/concepts/services-networking/dns-pod-service.md (hunk 4 above)
Comment (TinySong) [reply content/zh/docs/concepts/services-networking/dns-pod-service.md]: I think  `部分限定名称`  is a better understanding of the means of domains.

**Comment by TinySong** [comment]
> This PR seems to be conflicted with #33993, maybe you can check if this file have other place need to update.
> 
> * [[zh]Update content/zh/docs/concepts/services-networking/dns-pod-service.md #33993](https://github.com/kubernetes/website/pull/33993)

@Sea-n Thanks for pointing it out 😄 👍 .  already Reformatted the document format.

File: content/zh/docs/concepts/services-networking/dns-pod-service.md [hunk 7]
```diff
@@ -591,19 +588,21 @@ a list of search domains of up to 2048 characters.
...
-  后缀（例如：`mydns.svc.cluster.local`）。 Windows 可以解析能够使用此唯一后缀解析的
-  FQDN、服务与网络名字。
-  例如，在 `default` 命名空间中产生的某个 Pod 将具有 DNS 后缀 `default.svc.cluster.local`。
-  在 Windows Pod 中，你可以解析 `kubernetes.default.svc.cluster.local`
-  和 `kubernetes`，但无法解析部分限定名称（`kubernetes.default` 或
-  `kubernetes.default.svc`）。
+  Windows 将所有带有 `.` 的名称视为全限定域名（FQDN）并跳过全限定域名（FQDN）解析。
+- 在 Windows 上，可以使用的 DNS 解析器有很多。
+  由于这些解析器彼此之间会有轻微的行为差别，
+  建议使用 [`Resolve-DNSName`](https://docs.microsoft.com/powershell/module/dnsclient/resolve-dnsname)
```
Comment (Sea-n) [inline content/zh/docs/concepts/services-networking/dns-pod-service.md:597]: The long link can be list in a separate line.
```suggestion
  由于这些解析器彼此之间会有轻微的行为差别，建议使用
  [`Resolve-DNSName`](https://docs.microsoft.com/powershell/module/dnsclient/resolve-dnsname)
```

**Review by Sea-n — COMMENTED** [review]
Are you just overwrite with your version? Maybe you have to check each sentence and only modify only the necessary ones.

File: content/zh/docs/concepts/services-networking/dns-pod-service.md [hunk 8]
```diff
@@ -571,10 +571,7 @@ a list of search domains of up to 2048 characters.
 
 <!--
 ## DNS resolution on Windows nodes {#dns-windows}
--->
-## Windows 节点上的 DNS 解析 {#dns-windows}
```
[the remark this answers is not in the document]
Comment (TinySong) [reply content/zh/docs/concepts/services-networking/dns-pod-service.md:575]: yes, you are right, But this is only one headline， so I think separate or not is fine.

File: content/zh/docs/concepts/services-networking/dns-pod-service.md [hunk 9]
```diff
@@ -591,19 +588,21 @@ a list of search domains of up to 2048 characters.
   and `kubernetes`, but not the partially qualified names (`kubernetes.default` or
   `kubernetes.default.svc`).
 -->
+## Windows 节点上的 DNS 解析 {#dns-windows}
+
 - 在 Windows 节点上运行的 Pod 不支持 ClusterFirstWithHostNet。
-  Windows 将所有带有 `.` 的名字视为 FQDN 并跳过 FQDN 解析。
-- 在 Windows 上，可以使用多个 DNS 解析器。由于这些解析器的行为上略有不同，建议使用
```
Comment (TinySong) [inline content/zh/docs/concepts/services-networking/dns-pod-service.md:596]: `可以使用多个 DNS 解析器` make me confused.  I think `可以使用的 DNS 解析器有很多` is better.

File: content/zh/docs/concepts/services-networking/dns-pod-service.md [hunk 10]
```diff
@@ -591,19 +588,21 @@ a list of search domains of up to 2048 characters.
...
-  和 `kubernetes`，但无法解析部分限定名称（`kubernetes.default` 或
-  `kubernetes.default.svc`）。
+  Windows 将所有带有 `.` 的名称视为全限定域名（FQDN）并跳过全限定域名（FQDN）解析。
+- 在 Windows 上，可以使用的 DNS 解析器有很多。
+  由于这些解析器彼此之间会有轻微的行为差别，
+  建议使用 [`Resolve-DNSName`](https://docs.microsoft.com/powershell/module/dnsclient/resolve-dnsname)
+  powershell cmdlet 进行名称查询解析。
+- 在 Linux 上，有一个 DNS 后缀列表，当解析全名失败时可以使用。
+  在 Windows 上，你只能有一个 DNS 后缀，
+  及与该 Pod 的命名空间相关联的 DNS 后缀（例如：`mydns.svc.cluster.local`）。
```
[the remark this answers is not in the document]
Comment (TinySong) [reply content/zh/docs/concepts/services-networking/dns-pod-service.md:601]: that was my mistake.

File: content/zh/docs/concepts/services-networking/dns-pod-service.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (TinySong) [reply content/zh/docs/concepts/services-networking/dns-pod-service.md:575]: The other problem is primarily a document format, reference: https://kubernetes.io/zh/docs/contribute/localization_zh/#%E6%8E%92%E7%89%88%E6%A0%BC%E5%BC%8F

**Pull request merged**